deploy/charts/rook-ceph/templates/clusterrole.yaml

```diff
@@ -389,8 +389,6 @@ rules:
   - get
   - list
 ---
-# TODO: remove this, once https://github.com/rook/rook/issues/10141
```

Comment:
This clusterRole for getting nodes is now required for cephfs read affinity feature.
I've modified read affinity docs to indicate the same too.